### DISCUSSION TITLE
just overall a laggy mess

### DISCUSSION THREAD

**Discussion by DabDatBass** [ScratchAddons/ScratchAddons#760] [Discussion]
ScratchAddons is really useful, but I had to uninstall it because it lagged my computer a lot. If you can, please make it less laggy for users that have slower computers _(Chromebook, Raspberry Pi, etc)_? It would really help :)

**Comment by TheColaber** [comment]
Hey Glad to see you on gh! Is there a specific addon? Make sure your version is set to the latest!

**Reply by DabDatBass** [reply]
yes it was

**Reply by TheColaber** [reply]
Which addon causes lag?

**Reply by DabDatBass** [reply]
from loading bar to dev tools

**Reply by TheColaber** [reply]
WDYM? Which addon, the loading bar or devtools? Or both?

**Reply by GrahamSH-LLK** [reply]
First, make sure you were on the _latest_ version. It solves lots of lag problems. Then, turn off all addons, and just use Devtools. Does it still happen?

**Comment by WorldLanguages** [comment]
If you're experiencing lag while specifically moving blocks in the Scratch editor, please know that issue has been fixed in the current prerelease and will hit the Chrome and Firefox stores within a week.
If not, could you be more specific on how you're experiencing/feeling the lag?
